script.js

```diff
@@ -1,48 +1,126 @@
 let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
-//ash says we can use this as a global object
+
 let gameState = ["", "", "", "", "", "", "", "", ""];
+let currentPlayer = "x";
+const tiles = document.querySelectorAll(".tile");
 
 function decideTurn(currentPlayer) {
```

Comment:
I missed this earlier, but there's no need to pass currentPlayer into decideTurn, since its being used as a global variable